src/NetAnalyzers/UnitTests/Microsoft.NetFramework.Analyzers/DoNotUseInsecureDtdProcessingXmlDocumentConstructedWithNoSecureResolverTests.cs

```diff
@@ -580,9 +703,58 @@ End Class
         }
 
         [Fact]
-        public async Task XmlDocumentAsFieldSetResolverToInsecureResolverInOnlyMethodShouldGenerateDiagnostics()
+        public async Task XmlDocumentSetResolversInDifferentBlockPost452ShouldGenerateDiagnostic()
```

Comment:
Test is named as "should generate diagnostic" but there is no expected diagnostic listed.

Reply:
Thanks for pointing out. Fixed.